packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3635,6 +3670,23 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
   }
 
   handleGotoValueSelectedColumnNameChanged(columnName: ColumnName): void {
+    const { model } = this.props;
+    let cursorRow;
+    if (this.grid) {
+      ({ cursorRow } = this.grid.state);
+    }
```

Comment:
You could just write this like:
```suggestion
    const cursorRow = this.grid?.state.cursorRow;
```